pontoon/base/static/js/translate.js

```diff
@@ -2151,7 +2151,11 @@ var Pontoon = (function (my) {
         submittedPluralForm = 0;
       }
 
-      return $.ajax({
+      if (self.XHRupdateOnServer) {
+        self.XHRupdateOnServer.abort();
```

Comment:
I think that abort() won't guarantee that server didn't already processed your call. Maybe it should be better if you we would (at the very top of updateOnServer()) check if there's any ongoing request to the backend and then return.